src/common/format/index.js

```diff
@@ -15,21 +15,22 @@ const fallbacks = {
   type: 'feature',
 };
 
-export default (templates = {}) => {
-  const renderer = (name) => {
-    const render = compile(templates[name] || defaults[name], helpers);
-    return values => render({ ...fallbacks, ...values });
-  };
+const renderer = (templates, name) => {
+  const render = compile(templates[name] || defaults[name], helpers);
+  return values => render({ ...fallbacks, ...values }).trim();
+};
 
-  const commit = renderer('commit');
-  const branch = renderer('branch');
-  const cmd = renderer('command');
+export default (templates = {}) => {
+  const branch = renderer(templates, 'branch');
+  const commit = renderer(templates, 'commit');
 
-  const command = values => cmd({
+  // eslint-disable-next-line no-underscore-dangle
+  const _command = renderer(templates, 'command');
```

Comment:
how about we call this `commandRenderer` so we do not need the `eslint-disable...`

Reply:
🤔 I used `cmd` before, but that seemed a bit cryptic. Using `commandRenderer` seems a bit odd too, because it is essentially the same kind of "thing" as the `branch` and `commit` values above, but these aren't called `*Renderer`. 🤷‍♂️ I honestly don't know… naming is hard. To me the current name didn't feel ideal, but the alternatives I could think of didn't either.

Reply:
Also fine with sticking with it :)